Formula/joplin-terminal.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class JoplinTerminal < Formula
+  desc "Note taking and to-do application with synchronization capabilities"
+  homepage "https://joplinapp.org/"
+  url "https://registry.npmjs.org/joplin/-/joplin-2.2.2.tgz"
+  sha256 "15b932b51632dbcf8dfa0c39541af49b7e072b1a7c204e304d0fbdea6f1d787a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "Joplin CLI", shell_output("#{bin}/joplin version")
```

Comment:
We normally don't like tests that just do `version`. Given the circumstances, I think I'm fine with this -- however, can we add a comment documenting why we're only doing a `version` test here?

We can probably also be a bit stricter about matching the output since `joplin version` should also print the version.

Reply:
@carlocab Can I access the version as a variable here? I wanted to avoid having a fragile test that would require manual update on each version change.

I'll add a comment explaining why this is the test in use.

Reply:
Yes. The formula version is available as `version`.